### ISSUE TITLE
 Anti-proteinase 3 (PR3) antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6599]
**Preferred term label:**  Anti-proteinase 3 (PR3) antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
PR3 directed c-ANCA antibodies are present in 80-90% of granulomatosis with polyangiitis, 20-40% of microscopic polyangiitis, 20-40% of pauci-immune crescentic glomerulonephritis and 35% of eosinophilic granulomatosis with polyangiitis.

**Parent term (use hpo.jax.org/app)**
Cytoplasmic antineutrophil antibody positivity HP:0032230

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Granulomatosis With Polyangiitis OMIM:608710, ORPHA:900
Microscopic Polyangiitis ORPHA:727
Eosinophilic granulomatosis with polyangiitis ORPHA:183

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
